examples/buttons.html

```diff
@@ -0,0 +1,25 @@
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Buttons - Tachyons Tutorial</title>
+    <!-- Pure CSS is a minimalist CSS file we have included to make things look nicer -->
```

Comment:
Is this comment supposed to be here? Or is the `Pure CSS` file missing? (same with the other 3 html files)